ext/pcre/config0.m4

```diff
@@ -35,7 +35,7 @@ PHP_ARG_WITH(pcre-jit,,[  --with-pcre-jit         Enable PCRE JIT functionality
       if test -x "$PCRE2_CONF"; then
         AC_MSG_CHECKING(for PCRE2 10.30 or greater)
         PCRE2_VER=`$PCRE2_CONF --version`
-        if test "`echo $PCRE2_VER | sed 's,\.,,g'`" -lt 1030; then
+        if test "`echo $PCRE2_VER | $SED 's/\([[0-9]]*\)\.\([[0-9]]*\)\(.\)*/\1\2/g'`" -lt "1030"; then
```

Comment:
I think the first two `*` should be `+` here. Also, why `\(.\)*` rather than just `.*`?

Reply:
The `+` would work if `sed -r` would be used (regular expressions) which is on MacOS and FreeBSD `sed -E` option:
http://www.grymoire.com/Unix/Sed.html#uh-4a

The last asterisk has been fixed and simplified into `.*`...